app/src/main/java/com/example/roommatefinder/ProfileActivity.java

```diff
@@ -0,0 +1,68 @@
+package com.example.roommatefinder;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+
+public class ProfileActivity extends AppCompatActivity {
+
+    private ImageView profilePic;
+    private TextView profileName, profileClass, profileEmail, profileGender;
+    private Button profileUpdate;
+
+    //1st step: import firebase auth and database
```

Comment:
Your comments here are really useful to help me understand what is happening, though maybe adding a small blurb on why you have to do it this way would be helpful (since it is a new api we are using and since I wasn't part of implementing this step I don't necessarily know why some things are necessary)